JSON_Schemas/Weights_and_Condition_Scores/iscConditionScore.json

```diff
@@ -0,0 +1,29 @@
+{
+    "required": [
+    ],
+
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "iscScoreEventCore.json"
+        },
+        {
+            "properties": {
+                "conditionScoreReference": {
+                    "type": "object",
+                    "$ref": "iscScoreReference.json",
+                    "description": "The scheme or scoring reference guideline being used"
+                },
+                "score":{
+                    "type": "number",
+                    "description": "Value of the condition score"
```

Comment:
Add to description - Body condition scoring is a visual assessment of the amount of fat and muscle covering the bones of a cow, regardless of body size

Reply:
Added description.